quarkus/runtime/pom.xml

```diff
@@ -727,4 +727,30 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>product</id>
+            <activation>
+                <property>
+                    <name>skipProprietary</name>
```

Comment:
Is there some reason we need this property name? Can we just use `-Dproduct` or `-Pproduct` similarly as we used it before?

Reply:
I thought it could be helpful for the community builds as users could specify the property as it's more specific than the product profile. However, when I'm thinking about it right now, it could be sufficient to use only the `product` profile as the property was supposed to be an internal one anyway.

Reply:
I've removed it.